### ISSUE TITLE
FFI-Platypus-0.77_01/02: "dlrun.exe has stopped working"

### ISSUE THREAD

**Issue by twata1** [PerlFFI/FFI-Platypus#108]
Hi,

I noticed a little dialog box appeared when I run the following command in Strawberry Perl 5.14.4 on Windows 7 (32bit).

```cpan PLICEASE/FFI-Platypus-0.77_01.tar.gz```

or

```cpan PLICEASE/FFI-Platypus-0.77_02.tar.gz```

[Example]
```
C:\home\heaven>cpan PLICEASE/FFI-Platypus-0.77_02.tar.gz
```
(snip)

```
PR type.wint_t.size=2
PR probe.abi=1
PR probe.alloca=1
PR probe.bigendian=0
PR probe.bigendian64=0
```
(A little dialog box appears that shows "dlrun.exe has stopped working")

I don't know if it is a problem to ignore.

Thank you,

**Comment by plicease** [comment]
The platypus built should be fine, the message indicates that the probe program failed (it's supposed to fail, but not in such a vocal way), however, this definitely needs to be resolved because such a diagnostic is annoying and unseemly!  There is a similar diagnostic on Linux, although no GUI interaction, so a little less annoying.

**Comment by plicease** [comment]
I think #109 should resolve this, but can you try this:

```
C:\> set FFI_PLATYPUS_PROBE_OVERRIDE=bigendian=0:bigendian64=0
C:\> perl Makefile.PL
C:\> make
```

**Comment by plicease** [comment]
hrm. Actually I just was able to reproduce this with the complex probe.  I think the problem might actually be there.

**Comment by plicease** [comment]
I believe this should be fixed in 0.77_03 headed to cpan now.  If you have a chance, if you could please try it and let me know.

**Comment by twata1** [comment]
Under Strawberry Perl 5.14.4 on Windows 7, I certainly confirmed a little dialog box didn't appear when I run ```cpan PLICEASE/FFI-Platypus-0.77_03.tar.gz```.

Thank you for your help.
